dist/stat_data.py

```diff
@@ -335,6 +339,41 @@ def __init__(self, name, desc, flags=''):
     CapacityStat('fsync_all_fh_total', 'background fsync file handles considered'),
     CapacityStat('fsync_all_time', 'background fsync time (msecs)', 'no_clear,no_scale'),
 
+    ##########################################
+    # Checkpoint statistics
+    ##########################################
+    CheckpointStat('checkpoints', 'transaction checkpoints'),
```

Comment:
Not sure if this is just a me thing, but this statistic wasn't the clearest to me. Something like 'number of checkpoints' would have been clearer to me but I'm also happy to leave it as it is.

Reply:
No, this is not clear at all. The name is okay, but the description is incomplete. Is it (without reading the code further down) "number of checkpoints performed" or "current checkpoint state" or "transaction checkpoint calls"? Depending on what it is, try to make the description consistent with other similar stats.

Reply:
@korteland I also see that you did not add/change this, only moved it. So ignore this.

Reply:
I may as well improve it while I'm here. I'll use "number of checkpoints started" unless someone disagrees.